specification/datalake-analytics/resource-manager/Microsoft.DataLakeAnalytics/stable/2016-11-01/account.json

```diff
@@ -1564,6 +1762,21 @@
     }
   },
   "definitions": {
+    "ErrorContract": {
+      "title": "Error details.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "allOf": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
```

Comment:
>ErrorResponse [](start = 98, length = 13)

v2 error response already defines the `error` property at the top level ([https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json)), which would make this look like the following in the json:
```json
"error": {
  "error": {
    ...
  }
}
```

Instead of referencing `#/definitions/ErrorContract` you should reference `../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse` in all the default responses.

Reply:
fixed.